snuba/consumer.py

```diff
@@ -60,23 +65,10 @@ def process_message(
             ),
         )
 
-        if processed is None:
-            return None
-
-        if processed.action not in set(
-            [ProcessorAction.INSERT, ProcessorAction.REPLACE]
-        ):
-            raise InvalidActionType("Invalid action type: {}".format(processed.action))
```

Comment:
This is still validated in `flush_batch` to make sure that each result value is one of the expected types.